pyproject.toml

```diff
@@ -123,3 +123,16 @@ target-version = ['py38']
 [tool.check-wheel-contents]
 # W009: Wheel contains multiple toplevel library entries
 ignore = "W009"
+
+[tool.ruff]
+src = ["pytest"]
+select = [
+    "I",  # isort
+]
+
+[tool.ruff.lint.isort]
+force-single-line = true
+known-local-folder = ["pytest", "_pytest"]
+known-third-party = ["test_source", "test_excinfo"]
+force-sort-within-sections = true
+case-sensitive = false
```

Comment:
This is the default, would remove.

Reply:
aa5f90166  (Changing to true does nothing by the way.)